pkg/generator/steps/layout/tpl-new/py-services/_service_/__main__.py.tpl

```diff
@@ -0,0 +1,11 @@
+import {{.ServiceName}}.generated
+from {{.ServiceName}}.generated.app.mify_app import MifyServiceApp
+
+from libraries.generated.logs.logger import MifyLoggerWrapper
```

Comment:
Is this import required here? (also {{.ServiceName}}.generated) They are not used in this file. There could be a problem if a user is using a strict linter. He will throw errors for each unused import. And one more thing: maybe it will be a good idea to scan generated service with linter with strict rules.